Jenkinsfile

```diff
@@ -0,0 +1,52 @@
+properties([[$class: 'BuildDiscarderProperty',
+            strategy: [$class: 'LogRotator', artifactDaysToKeepStr: '', artifactNumToKeepStr: '1', daysToKeepStr: '60', numToKeepStr: '60']],
+            [$class: 'RebuildSettings', autoRebuild: false, rebuildDisabled: false],
+            [$class: 'ParametersDefinitionProperty', parameterDefinitions: [
+            [$class: 'StringParameterDefinition', defaultValue: '6.0-SNAPSHOT', description: 'Product version to build', name: 'NUXEO_VERSION'],
+            [$class: 'StringParameterDefinition', defaultValue: '', description: 'Optional - Use the
+specified URL (eg a link to staging) as the source for the distribution instead of maven', name: 'DISTRIBUTION_URL'],
```

Comment:
Reword `DISTRIBUTION_URL` description to:
> Optional - Alternate URL to download the distribution from instead of the default Maven artifact download.
> For instance: http://community.nuxeo.com/static/snapshots/nuxeo-server-tomcat-10.3-SNAPSHOT.zip

It seems there's a line return here :/